scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -70,17 +71,21 @@
     </div>
     <div class="row">
         <div class="col-sm">{{ inheritance_panel() }}</div>
-        <div class="col-sm">{{ omim_phenotypes() }}</div>
+        <div class="col-sm">{{ omim_phenotypes(variant) }}</div>
     </div>
     <div class="row">
-      <div class="col-sm">{{ frequency_list() }}</div>
-      <div class="col-sm">{{ severity_list() }}</div>
+      <div class="col">{{ frequencies(variant) }}</div>
+      {% if config['LOQUSDB_SETTINGS'] %}
+        <div class="col">{{ observations_panel(variant, observations) }}</div>
+      {% endif %}
+      <div class="w-100"></div>
```

Comment:
👀

Reply:
Causes this I suppose?
![Screenshot 2020-02-05 at 16 55 24](https://user-images.githubusercontent.com/758570/73860614-170a2580-483c-11ea-82ba-bd78966d15cb.png)

Reply:
:)